arm/Microsoft.Insights/privateLinkScopes/.parameters/parameters.json

```diff
@@ -2,7 +2,7 @@
     "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
-        "privateLinkScopeName": {
+        "name": {
```

Comment:
May I hijack this PR for renaming this resource and better align to our naming? 
`sxx-az-pls-x-001`

Reply:
done